content/influxdb/cloud-dedicated/admin/custom-partitions/best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Partitioning best practices
+description: >
+  Learn best practices for applying custom partition strategies to your data
+  stored in InfluxDB.
+menu:
+  influxdb_cloud_dedicated:
+    name: Best practices
+    parent: Manage data partitioning
+weight: 202
+---
+
+Use the following best practices when defining custom partitioning strategies
+for your data stored in {{< product-name >}}.
+
+- [Partition by tags that you commonly query for a specific value](#partition-by-tags-that-you-commonly-query-for-a-specific-value)
+- [Only partition by tags that _always_ have a value](#only-partition-by-tags-that-always-have-a-value)
+- [Avoid over-partitioning](#avoid-over-partitioning)
+
+## Partition by tags that you commonly query for a specific value
+
+Custom partitioning primarily benefits queries that look for a specific tag
+value in the `WHERE` clause. For example, if you often query data related to a
+specific ID, partitioning by the tag that stores the ID helps the InfluxDB
+query engine to more quickly identify what partitions contain the relevant data.
+
+{{% note %}}
+#### Be careful partitioning on high-cardinality tags
+
+Partitioning using tags with many (500K+) unique values can actually hurt
```

Comment:
@NGA-TRAN I just used the `500K+` number as a placeholder and don't have any evidence to suggest this is an accurate range for "high-cardinality" tags. Do you have any guidance here?

Reply:
If you really want a number, use `10K+`.

Reply:
Thank you! It's not so much that _I_ want a number, but I know our customers will 😄 and will ask for it.